imageregistry/v1/00-crd.yaml

```diff
@@ -0,0 +1,555 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: configs.imageregistry.operator.openshift.io
+spec:
+  group: imageregistry.operator.openshift.io
+  scope: Cluster
+  version: v1
+  names:
+    kind: Config
+    listKind: ConfigList
+    plural: configs
+    singular: config
+  subresources:
+    status: {}
+  validation:
+    openAPIV3Schema:
+      description: Config is the configuration object for a registry instance managed
+        by the registry operator
+      properties:
+        apiVersion:
+          description: 'APIVersion defines the versioned schema of this representation
+            of an object. Servers should convert recognized schemas to the latest
+            internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+          type: string
+        kind:
+          description: 'Kind is a string value representing the REST resource this
+            object represents. Servers may infer this from the endpoint the client
+            submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+          type: string
+        metadata:
+          type: object
+        spec:
+          properties:
+            defaultRoute:
+              description: DefaultRoute indicates whether an external facing route
+                for the registry should be created using the default generated hostname.
+              type: boolean
+            disableRedirect:
+              description: DisableRedirect controls whether to route all data through
+                the Registry, rather than redirecting to the backend.
+              type: boolean
+            httpSecret:
+              description: HTTPSecret is the value needed by the registry to secure
+                uploads, generated by default.
+              type: string
+            logging:
+              description: LogLevel determines the level of logging enabled in the
```

Comment:
should start with `logging` vs. `LogLevel`, right?

Reply:
Yes, indeed. After `make update` it got solved.